pkg/util/emailutil/email.go

```diff
@@ -28,7 +29,11 @@ func SendMail(ctx context.Context, emailAddr string, header string, body string)
 	m.SetBody("text/html", body)
 
 	d := gomail.NewDialer(host, port, email, password)
-	d.TLSConfig = &tls.Config{InsecureSkipVerify: true}
+	if sSLEnable {
+		d.TLSConfig = &tls.Config{InsecureSkipVerify: false}
```

Comment:
no need if else, 
d.TLSConfig = &tls.Config{InsecureSkipVerify: !sslEnable} will be better.